vcpkg-registry/ports/cryptoutils/portfile.cmake

```diff
@@ -47,6 +47,6 @@ vcpkg_cmake_configure(
 )
 
 vcpkg_cmake_install()
-
+file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```

Comment:
We could a bit simplify it using [`vcpkg_install_copyright()`](https://github.com/microsoft/vcpkg/blob/9bf374fb6d99f8973ec8781d3b41ffac2defee63/docs/maintainers/vcpkg_install_copyright.md) as the documentation [suggests](https://github.com/microsoft/vcpkg/blob/9bf374fb6d99f8973ec8781d3b41ffac2defee63/docs/maintainers/maintainer-guide.md#install-copyright-file).

Reply:
Ah! Good point - I just copied an existing port - well found.

Reply:
There you go :-)